.env

```diff
@@ -1,5 +1,5 @@
 VUE_APP_CAPTCHA_SITE_KEY=6LeZgrgUAAAAACAeeUO1yZhMWqO30BgcBppjPnz4
-VUE_APP_API_URL=http://921e51dd.ngrok.io/api
+VUE_APP_API_URL=https://localhost:5001/api
 VUE_APP_DEFAULT_LAYOUT=Default
 VUE_APP_TITLE=Quizyfy
 NODE_ENV=development
```

Comment:
Everything should be in .env.example

I can see your keys and other stuff.
Just set the defaults in .env.example and in api keys etc. put dummy data.
.env should not be commited. Exclude it in .gitignore, instead commit .env.example

Reply:
Also project should include readme so other developers that are participating in your project can setup their environments easily.

Reply:
I addressed the first issue in 91c42981782515eeae5c085b69540b5916a1c33e

Reply:
The second one should be in different PR i think.